spec/features/post_spec.rb

```diff
@@ -91,16 +91,15 @@
 
     scenario 'Should show the post commentor name ' do
       visit user_post_path(@user1, @post1)
-      comment = @post1.comments.first
-      expect(page).to have_content(comment.author.name)
+      expect(page).to have_content(@comment1.user.name)
     end
 
-    scenario 'Should show the comment each comment author left ' do
-      visit user_post_path(@user1, @post1)
-      comment = @post1.comments.first
-      comment2 = @post1.comments.last
-      expect(page).to have_content(comment.text)
-      expect(page).to have_content(comment2.text)
+    scenario 'Should show the comment each comment user left ' do
+      @user = User.first
+      @post = Post.first
+      visit user_post_path(@user, @post)
+      expect(page).to have_text(@post.comments[0].text)
+      expect(page).to have_text(@post.comments[1].text)
```

Comment:
- [ ] Hello and nice job on adding the tests but this test suit is failing due to the wrong argument provided, `hint: please you are trying to get a comment of a specific user in the post, not the post comment`

![Capture d’écran 2022-08-30 145935](https://user-images.githubusercontent.com/61562626/187457589-e79eec00-615e-4601-bce5-4c7693a04d76.png)